ruby/ql/lib/codeql/ruby/frameworks/StandardLibrary.qll

```diff
@@ -335,3 +336,18 @@ class ModuleEvalCallCodeExecution extends CodeExecution::Range, DataFlow::CallNo
 
   override DataFlow::Node getCode() { result = this.getArgument(0) }
 }
+
+/** Flow summary for `Regexp.escape`. */
```

Comment:
Not one for this PR, but I wonder if we ought to split out `StandardLibrary.qll` soon? Maybe something like:

```
ql/lib/codeql/ruby/frameworks
├── core
│   ├── Kernel.qll
│   └── Object.qll
│   └── Regexp.qll
└── std-lib
    ├── Date.qll
    └── Open3.qll
```